client/src/pages/ReviewCreate.jsx

```
@@ -1,7 +1,181 @@
+          </div>
+          <div className={styles.reviewWrapper}>
+            <div className={styles.wrapper}>
+              <h2>도서</h2>
+              <BookDashboard bookDetail={bookDetail}></BookDashboard>
+            </div>
+            <div className={styles.wrapper}>
+              <div className={styles.flexBox}>
+                <h2>이미지</h2>
+                <div className={styles.flexBox} onClick={deleteFileHandler}>
+                  <img src={deleteFileIcon} alt="deleteFile" className={styles.icon} />
+                  <span className={styles.grayText}>이미지 삭제하기</span>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)